docs/examples/index.md

```diff
@@ -1,8 +1,134 @@
+
+const input = { hashtags: ['twitter'], resultsLimit: 20 };
+
+// Run the actor and wait for it to finish up to 60 seconds.
+// Input is not persisted for next runs.
+const runData = await actorClient.call(input, { waitSecs: 60 });
+```
+
+To run multiple inputs with the same Actor, most convenient way is to create multiple [tasks](https://docs.apify.com/platform/actors/running/tasks) with different inputs. Task input is persisted on Apify platform when task is created.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const socialsHashtags = ['facebook', 'twitter', 'instagram'];
+
+// Multiple input schemas for one Actor can be persisted in tasks.
+// Tasks are saved in the Apify platform and can be run multiple times.
+const socialsTasksPromises = socialsHashtags.map((hashtag) => client.tasks().create({
+    actId: 'apify/instagram-hashtag-scraper',
+    name: `hashtags-${hashtag}`,
+    input: { hashtags: [hashtag], resultsLimit: 20 },
+    options: { memoryMbytes: 1024 },
+}));
+
+// Create all tasks in parallel
+const createdTasks = await Promise.all(socialsTasksPromises);
+
+// Run all tasks in parallel
+await Promise.all(createdTasks.map((task) => client.task(task.id).call()));
+```
+
+## Getting latest data from an Actor, joining datasets
+
+Actor data are stored to [datasets](https://docs.apify.com/platform/storage/dataset). Datasets can be retrieved from Actor runs. Dataset items can be listed with pagination. Also, datasets can be merged together to make analysis further on with single file as dataset can be exported to various data format (CSV, JSON, XSLX, XML). [Integrations](https://docs.apify.com/platform/integrations) can do the trick as well.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+const actorRuns = actorClient.runs();
+
+// See pagination to understand how to get more datasets
+const actorDatasets = await actorRuns.list({ limit: 20 });
+
+const mergingDataset = await client.datasets().getOrCreate('merge-dataset');
+
+for (const datasetItem of actorDatasets.items) {
+    // Dataset items can be handled here. Dataset items can be paginated
+    const datasetItems = await client.dataset(datasetItem.defaultDatasetId).listItems({ limit: 1000 });
+
+    // Items can be pushed to single dataset
+    await client.dataset(mergingDataset.id).pushItems(datasetItems.items);
+
+    // ...
+}
+```
+
+## Handling webhooks
+
+[Webhooks](https://docs.apify.com/platform/integrations/webhooks) can be used to get notifications about Actor runs. For example, a webhook can be triggered when an Actor run finishes successfully. Webhook can receive dataset ID for further processing.
```

Comment:
I'd also describe the use case here and for sure explain what is express, and what it does in this example - there is nothing about it here now.